Add release notes for 1.18

xprazak2 opened the pull request.

Commit:
Add release notes for 1.18

tbrisker commented on _includes/manuals/1.18/1.2_release_notes.md:
```diff
@@ -1,15 +1,288 @@
+Hosts can now be provisioned with RancherOS. ([#15634](http://projects.theforeman.org/issues/15634), [0ace99ee](http://github.com//foreman/commit/0ace99ee2cc11cfa4e0a7d69be9a411b83812e68), [5fc1b19a](http://github.com//foreman/commit/5fc1b19ae5d95ecd42379f46847ff4240bd618e9), [8c837777](http://github.com//community-templates/commit/8c83777715c1b7af9ec45e62332368a42e83ac0c))
+
+
+#### Anonymize Audit logs
```
Perhaps combine this with the deletion as well (#22158)?

tbrisker commented on _includes/manuals/1.18/1.2_release_notes.md:
```diff
@@ -1,15 +1,288 @@
+
+#### Web Interface
+ * Bump version of patternfly-react to support inline edits ([#23462](http://projects.theforeman.org/issues/23462), [f927833c](http://github.com//foreman/commit/f927833c5dafa6859afe009e29e2a04c8f7ae231), [e5ca4f3e](http://github.com//foreman/commit/e5ca4f3e05b4d0bec21478fa1cc72ff57befcc9b))
+ * Add redux container for breadcrumb switcher ([#22855](http://projects.theforeman.org/issues/22855), [833e2aef](http://github.com//foreman/commit/833e2aef24d6f3cb6eae90213ea198dd354a5709))
```
I think the breadcrumbs and resource switcher also deserve a headline

tbrisker commented on _includes/manuals/1.18/1.2_release_notes.md:
```diff
@@ -1,15 +1,288 @@
+#### Compute resources - VMware
+ * VMware: Exception listing VMs in the root of the datacenter ([#22798](http://projects.theforeman.org/issues/22798), [68e96c10](http://github.com//foreman/commit/68e96c10b596d67106a3ed49535231019d8507b4))
+ * Rescue from errors on per-host basis when associating vmware VMs ([#21317](http://projects.theforeman.org/issues/21317), [7af7f44c](http://github.com//foreman/commit/7af7f44c3324bb93f2402b2cb528c23b5ccc5b06))
+ * Failed to update a compute vmware (VMware) instance node.test.domain: undefined method `[]' for nil:NilClass ([#19990](http://projects.theforeman.org/issues/19990), [1dbb2c5b](http://github.com//foreman/commit/1dbb2c5b319dbab023f3aaa98019d7b9cd0ee8f6))
```
I think the backtick here needs escaping for the markdown formatting to be correct

tbrisker reviewed (changes requested):
Added a few inline comments, otherwise looks good!

xprazak2 commented:
I added breadcrumbs and cleaning audits into headline features, fixed the markup.

I will add contributors after we've tagged, our script uses the tags to generate the list.

xprazak2 commented:
I added contributor list.

mmoll commented:
All the links should go directly to https://

mmoll commented on _includes/manuals/1.18/1.2_release_notes.md:
```diff
@@ -1,22 +1,302 @@
+
+#### RancherOS support
+
+Hosts can now be provisioned with RancherOS. ([#15634](http://projects.theforeman.org/issues/15634), [0ace99ee](http://github.com//foreman/commit/0ace99ee2cc11cfa4e0a7d69be9a411b83812e68), [5fc1b19a](http://github.com//foreman/commit/5fc1b19ae5d95ecd42379f46847ff4240bd618e9), [8c837777](http://github.com//community-templates/commit/8c83777715c1b7af9ec45e62332368a42e83ac0c))
```
Maybe https://rancher.com/rancher-os/ could get linked here

mmoll commented on _includes/manuals/1.18/1.2_release_notes.md:
```diff
@@ -1,22 +1,302 @@
+
+#### Add MTU support for subnets
+
+MTU for subnets is now supported. ([#21694](http://projects.theforeman.org/issues/21694), [f175f751](http://github.com//foreman/commit/f175f7510d9e334c468ecfdcfc69c7a8485f21a1), [92c17a1a](http://github.com//foreman/commit/92c17a1a1ff6f6ed2d94b5ae13279634fa5127f2), [6b65706a](http://github.com//foreman/commit/6b65706af1b0604ddab98774c00391ccddea1fb7), [d313cc81](http://github.com//community-templ ...
```
Maybe a tad better: "Setting the MTU per subnet is now supported. The default will be 1500 (Ethernet default value)." or similar. Otherwise, without knowing what this change is about it's hard to understand at the first read, IMHO.

xprazak2 commented:
The links go to https:// now and I added a link for RancherOS

tbrisker commented on _includes/manuals/1.18/1.2_release_notes.md:
```diff
@@ -1,22 +1,302 @@
+
+Hosts can now be provisioned with [RancherOS](https://rancher.com/rancher-os/). ([#15634](https://projects.theforeman.org/issues/15634), [0ace99ee](https://github.com//foreman/commit/0ace99ee2cc11cfa4e0a7d69be9a411b83812e68), [5fc1b19a](https://github.com//foreman/commit/5fc1b19ae5d95ecd42379f46847ff4240bd618e9), [8c837777](https://github.com//community-templates/commit/8c83777715c1b7af9ec45e6233 ...
+
+#### Anonymize Audit logs
```
This heading should also be changed, something like "Anonymize or clean audit logs"

xprazak2 commented:
Updated.

tbrisker reviewed (approved):
Thanks @xprazak2 !
